test/parallel/test-http-set-cookies.js

```diff
@@ -72,14 +70,12 @@ server.on('listening', function() {
     });
 
     res.on('end', function() {
-      if (++nresponses === 2) {
-        server.close();
-      }
+      countdown.dec();
     });
   });
 
 });
 
 process.on('exit', function() {
-  assert.strictEqual(2, nresponses);
+  assert.strictEqual(countdown.remaining, 0);
```

Comment:
The `process.on('exit')` can be removed as `server.close()` needs to run before the process will exit.